### ISSUE TITLE
Improve Modals Accessibility 

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1561]
Certain modals are not accessible (the focus leaves the modal)
Modals targeted in this issue are those that use the following functions:
- UI.alert
- UI.prompt
